symfony/src/Entity/DeceasedLog.php

```diff
@@ -0,0 +1,125 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\DeceasedLogRepository")
+ */
+class DeceasedLog
+{
+    /**
+     * @ORM\Id()
+     * @ORM\GeneratedValue()
+     * @ORM\Column(type="integer")
+     */
+    private $id;
+
+    /**
+     * @ORM\Column(type="string", length=50)
+     */
+    private $participantId;
+
+    /**
+     * @ORM\Column(type="datetime")
+     */
+    private $insertTs;
+
+    /**
+     * @ORM\Column(type="datetime", nullable=true)
+     */
+    private $deceasedTs;
+
+    /**
+     * @ORM\Column(type="string", length=100, nullable=true)
+     */
+    private $hpoId;
```

Comment:
hpoId is a confusing column name. Don't we have places where hpoId is used as a legacy synonym for awardee ID? Shouldn't this be representing an organization ID instead?